spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/DestinationNode.java

```diff
@@ -56,7 +56,7 @@ public String stringify(boolean includePositionalInfo) {
 
 	@Override
 	public String toString() {
-		return getDestinationName();
+		return ":" + getDestinationName();
```

Comment:
ok, I am trying to understand what prompted this change as I don't seem to find a usage of this change with respect to this PR.